en/subscription_services_privacy_notice.md

```diff
@@ -1,55 +1,67 @@
+__Technical data.__ Across Subscription Services, your IP address is temporarily collected as part of our server logs for 90 days. For VPN, Mozilla also receives information about the installed version of your Mozilla VPN client software, and the devices they are installed on, including their operating systems and hardware configuration. We use Technical data to improve performance and stability for our users, and to measure the Services’ performance. 
+
+In addition, if you have provided consent to data collection within the application, our partner Sentry.io will receive technical data for error and performance monitoring purposes.  Learn more at [Sentry.io’s Privacy Policy](https://sentry.io/privacy/).
 
 __Location data.__ Mozilla Subscription Services receives your IP address when you sign up for and use the service(s) and it is collected as part of our servers logs for 90 days; we use the IP address in order to approximate your location because Mozilla Subscription Services availability, pricing and offers may depend on your country. We also use it for fraud and abuse prevention purposes across our products.
 
-__Network data.__ When you activate Mozilla VPN, it will encrypt your internet traffic and send it to Mullvad. No logs of your network activity are maintained by either Mozilla nor our partner Mullvad when you're using the Mozilla VPN service.  Learn more at [Mullvad’s Privacy Policy](https://mullvad.net/help/no-logging-data-policy/).
+__Network data.__ When you activate Mozilla VPN, it will encrypt your internet traffic and send it to Mullvad. No logs of your network activity are maintained by either Mozilla nor our partner Mullvad when you're using the Mozilla VPN service in any circumstance.  Learn more at [Mullvad’s Privacy Policy](https://mullvad.net/help/no-logging-data-policy/).
+
+
+### Mozilla Monitor {: #monitor }
+
+__Interaction data.__ Mozilla receives data about your interactions with the Services. For instance, when you log in and out and the preferences you set; learn more about [Monitor Interaction data](https://dictionary.telemetry.mozilla.org/apps/monitor_frontend). More generally, Mozilla may also use cookies, device information, and IP addresses, along with clear GIFs, cookies and third party services to help us understand in the aggregate how users engage with our products, services, communications, websites, online campaigns, snippets, devices, and other platforms. We use:
```

Comment:
Lower case `s` in `Services` when used along?
Lower case `i` in `Monitor Interation data`?